snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedTransientSnapshot.java

```diff
@@ -161,7 +189,7 @@ public String toString() {
         + ", checksum="
         + checksum
         + ", metadata="
```

Comment:
:wrench: to avoid confusion:

```suggestion
        + ", id="
```